plugins/net_plugin/net_plugin.cpp

```diff
@@ -1071,58 +1115,70 @@ namespace eosio {
-        elog( "take_chunk called, but sync_receiving is empty");
-        return;
-      }
-      sync_state_ptr ss;
-      c->sync_receiving.swap(ss);
-      fc_dlog(logger, "conn ${n} losing recv blks ${s} to ${e}",("n",c->peer_name() )("s",ss->start_block)("e",ss->end_block));
-
-      if( chain_plug->chain().head_block_num() == sync_known_lib_num ) {
-        handshake_message hello;
-        handshake_initializer::populate(hello);
-        fc_dlog(logger, "All caught up with last known last irreversible block resending handshake");
-        for( auto &ci : my_impl->connections) {
-          if( ci->current()) {
-            hello.generation = ++ci->sent_handshake_count;
-            fc_dlog(logger, "send to ${p}", ("p",ci->peer_name()));
-            ci->enqueue( hello );
-          }
+    if( !c->sync_receiving) {
+      elog( "take_chunk called, but sync_receiving is empty");
+      return;
+    }
+    sync_state_ptr ss;
+    c->sync_receiving.swap(ss);
+    fc_dlog(logger, "conn ${n} covered blks ${s} to ${e}",("n",c->peer_name() )("s",ss->start_block)("e",ss->end_block));
+
+    if( chain_plug->chain().head_block_num() == sync_known_lib_num ) {
+      handshake_message hello;
+      handshake_initializer::populate(hello);
+      fc_dlog(logger, "All caught up with last known last irreversible block resending handshake");
+      for( auto &ci : my_impl->connections) {
+        if( ci->current()) {
+          hello.generation = ++ci->sent_handshake_count;
+          fc_dlog(logger, "send to ${p}", ("p",ci->peer_name()));
+          ci->enqueue( hello );
         }
       }
-      if( c->connected()) {
-        assign_chunk( c);
-      }
-      request_next_chunk();
     }
+    request_next_chunk();
+  }
 
-    void sync_manager::start_sync( connection_ptr c, uint32_t target) {
-      if( !syncing()) {
-        sync_last_requested_num = chain_plug->chain().head_block_num();
-      }
-      ilog( "Catching up with chain, our last req is ${cc}, theirs is ${t}",
-           ( "cc",sync_last_requested_num)("t",target));
-      if( target > sync_known_lib_num) {
-        sync_known_lib_num = target;
-      }
-      if( c->sync_receiving && c->sync_receiving->end_block > 0) {
-        return;
+  void sync_manager::start_sync( connection_ptr c, uint32_t target) {
+    if( !syncing()) {
+      sync_last_requested_num = chain_plug->chain().head_block_num();
+      fc_dlog(logger, "Inform other open connections that we are syncing");
+
+      handshake_message hello;
+      handshake_initializer::populate(hello);
+      for( auto &ci : my_impl->connections) {
+        if( ci->current()) {
+          hello.generation = ++ci->sent_handshake_count;
+          fc_dlog(logger, "send to ${p}", ("p",ci->peer_name()));
+          ci->enqueue( hello );
+        }
       }
-      assign_chunk( c);
-      request_next_chunk();
     }
+    ilog( "Catching up with chain, our last req is ${cc}, theirs is ${t}", ( "cc",sync_last_requested_num)("t",target));
```

Comment:
fc_dlog(logger

Reply:
No, I want the ilog to be used to report all these occurrences. They should be infrequent.